edx_lint/pylint/annotations_check.py

```diff
@@ -427,6 +446,25 @@ def check_annotation_group(self, search, annotations, node):
                 line=line_number,
             )
 
+    @utils.check_messages(MISSING_ANNOTATION)
+    def visit_call(self, node):
+        """
+        Check for missing annotations.
+        """
+        toggle_func_names = ["WaffleFlag", "NonNamespacedWaffleFlag", "WaffleSwitch", "NonNamespacedWaffleSwitch"]
+        if not isinstance(node.func, Name) or node.func.name not in toggle_func_names or not node.args:
```

Comment:
The thing is that at linting time, we do not have access to the python objects. So we cannot write `isinstance(node.func, BaseToggle)`. In order to do that, we would have to import the actual python module, which would cause dependency issues.

Reply:
Right.  I was thinking of a try/except block around the import like https://github.com/edx/edx-django-utils/blob/master/edx_django_utils/monitoring/internal/utils.py#L22-L25.  You could add edx-toggles as a testing dependency in edx-lint, and make this check conditional on actually having performed the import.

Reply:
After discussion, realized that we don't have the class.  Marking resolved.